packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -352,27 +273,27 @@ if (idxContext != null)
     VerifyAuthenticatorAsync(verifyAuthenticatorOptions, idxContext);
 ```
 
-If the `otp` value is valid, the `AuthenticationStatus` property of the `AuthenticationResponse` object returned by `VerifyAuthenticatorAsync` is `Success`. In this case, call `AuthenticationHelper.GetIdentityFromTokenResponseAsync` to retrieve the OIDC claims information about the user and pass them into your application. The user has now signed in.
+If the user has not opened the magic link in the same browser, the `otp` value isn't valid, or an `AuthenticationStatus` is returned that is not handled by your case statement, you should advise the user to return to the original tab in the browser and enter the `otp` value there to proceed.
```

Comment:
```suggestion
If the user hasn't opened the magic link in the same browser, the `otp` value isn't valid, or an `AuthenticationStatus` is returned that isn't handled by your case statement, you should advise the user to return to the original tab in the browser and enter the `otp` value there to proceed.
```
Suggestion: This sentence was difficult to follow - had to read a few times. Maybe try this construction?

For the following conditions, advise the user to return to the original tab in the browser and enter the `otp` value to proceed:

- User hasn't opened the magic link in the same browser
- The `otp` value isn't valid
- An `AuthenticationStatus` is returned that isn't handled by your case statement

Reply:
Fixed.